Steps/RustSetupSteps.yml

```diff
@@ -79,13 +79,13 @@ steps:
 # those on both Linux and Windows agents for consistency in the pipeline runs.
 #
 - script: |
-    rustup install --no-self-update 1.84.0
-  displayName: Install Rust 1.84.0 (Windows)
+    rustup install --no-self-update 1.85.0

```

Comment:
@Javagedes, have these steps already been tested 1.85?

Reply:
I ran these steps locally and did not see any issues, if that is what you mean